Mykyta titar

Tituha opened the pull request:
Added Homework folders

Tituha committed:
Merge branch 'main' of https://github.com/kolyasalubov/Lv-14.03.PythonFundamentals into Mykyta_Titar

Tituha committed:
Merge branch 'main' into Mykyta_Titar

Tituha committed:
On branch Mykyta_Titar
    Changes to be committed:
         added a new Folder with screenshot of game
             new file:   HW2/screenshot_of_game.png

s

Tituha committed:
added 4 new folders for GitHub homework On branch Mykyta_Titar
    Changes to be committed:
        new file:   .idea/inspectionProfiles/profiles_settings.xml
	new file:   .idea/misc.xml
	new file:   .idea/vcs.xml
	new file:   HW1/First_Test.txt.py
	new file:   HW1/main (2).py
	renamed:    Zen.txt -> HW2/Zen.txt

Tituha committed:
retutn file Zen.txt

Tituha committed:
return back file Zen.txt

Tituha committed:
return back file Zen.txt
   On branch Mykyta_Titar

Tituha committed:
Did the third homework in one Folder:HW3, enjoy)

Tituha committed:
Merge branch 'main' into Mykyta_Titar

kolyasalubov commented on HW1/First_Test.txt.py:
```diff
@@ -0,0 +1,8 @@
+name = input("What is your name?:")
```
good job

kolyasalubov commented on HW1/main (2).py:
```diff
@@ -0,0 +1,6 @@
+A = int(input('Put first number:'))
```
good job

kolyasalubov commented on HW3/First_Github_Task.py:
```diff
@@ -0,0 +1,19 @@
+# First assignment
+
```
good job

kolyasalubov commented on HW3/Third_Github_Task.py:
```diff
@@ -0,0 +1,9 @@
+
```
good job

kolyasalubov commented on HW3/Second_Github_Task.py:
```diff
@@ -0,0 +1,18 @@
+# First assignment
```
please look at this implementation:

number = int(input("Enter your number: "))

one_div, remainder = divmod(number, 1000)
two_div, remainder = divmod(remainder, 100)
three_div, four_div = divmod(remainder, 10)

Item 1: find of product of the digits of this number:
prod_of_dig = one_div * two_div * three_div * four_div
print("Product of the digits of this number:", prod_of_dig)

Item 2: write the number in reverse order:
str_number = str(number)[::-1]
print("Reverse of our number:", str_number)

Item 3: in ascending order, you need to sort the number included in the given number:
sort_number = ''.join(sorted(str(number)))
print("Ascending order of digits in the number:", sort_number)

Tituha committed:
Added the fourth HW with temperature task
 On branch Mykyta_Titar
 Changes to be committed:
new file:   HW4/First_Github_Task.py

Tituha committed:
added HW4 and HW5 on branch Mykyta_Titar
Changes to be committed:
	modified:   HW4/First_Github_Task.py
	new file:   HW5/First_GitHub_Task.py
	new file:   HW5/Second_GitHub_Task.py
	new file:   HW5/Third_GitHub_Task.py

Tituha committed:
Added HW4, HW5 On branch Mykyta_Titar
Changes to be committed:
	modified:   HW5/Second_GitHub_Task.py
	modified:   HW5/Third_GitHub_Task.py

Tituha committed:
Added first task of HW6  On branch Mykyta_Titar

Changes to be committed:
	new file:   HW6/FIrst_GitHub_Task.py
	new file:   HW6/Second_GitHub_Task.py

kolyasalubov commented on HW4/First_Github_Task.py:
```diff
@@ -0,0 +1,8 @@
+celsius = float(input("Enter the temperature in Celcius:"))
```
good job

kolyasalubov commented on HW5/First_GitHub_Task.py:
```diff
@@ -0,0 +1,11 @@
+# Making list with floats from lists with integers
+
```
good job

kolyasalubov commented on HW5/Second_GitHub_Task.py:
```diff
@@ -0,0 +1,13 @@
+# Making fibonacci from user's number
+
```
good job

kolyasalubov commented on HW5/Third_GitHub_Task.py:
```diff
@@ -0,0 +1,7 @@
+# Making factogit rial from user's number
+
```
good job
look at this implementation:
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)

kolyasalubov commented on HW6/FIrst_GitHub_Task.py:
```diff
@@ -0,0 +1,15 @@
+int1 = range(1, 11)
```
good job

Tituha committed:
added second file of HW2  On branch Mykyta_Titar
 Changes to be committed:
	modified:   HW6/Second_GitHub_Task.py
	new file:   HW7/First_GitHub_Task.py

Tituha committed:
Merge branch 'main' of https://github.com/kolyasalubov/Lv-14.03.PythonFundamentals

Tituha committed:
Added seventh GitHub_Home_Work

kolyasalubov commented on HW6/Second_GitHub_Task.py:
```diff
@@ -0,0 +1,7 @@
+user_login = input("Enter your login please:")
```
Here's an optimized version of your code:

while True:
    user_login = input("Enter your login please: ")
    if user_login == "First":
        print("Thank you! We are glad to have you at our website, enjoy!")
        break
    else:
        print("Error! You wrote the wrong login. Please, try one more time.")

kolyasalubov commented on HW7/First_GitHub_Task.py:
```diff
@@ -0,0 +1,19 @@
+def largest_num(num1: int, num2: int) -> int:
```
good job

kolyasalubov commented on HW7/Second_GitHub_Task.py:
```diff
@@ -0,0 +1,31 @@
+def area_rectangle(length: int, width: int) -> int:
```
good job

kolyasalubov commented on HW7/Second_GitHub_Task.py:
```diff
@@ -0,0 +1,31 @@
+
+
+PI = 3.14
+finding = input("What kind of figure do you need the area for?:")
```
finding = input("What kind of figure do you need the area for?:").lower()

kolyasalubov commented on HW7/Second_GitHub_Task.py:
```diff
@@ -0,0 +1,31 @@
+while finding != "rectangle" or "triangle" or "circle":
+    another_try = input(f"Your figure printed incorrect. Please, try again instead of({finding}):")
+    finding = another_try
+    if finding.lower() == "rectangle":
```
if finding == "rectangle":

kolyasalubov commented on HW7/Second_GitHub_Task.py:
```diff
@@ -0,0 +1,31 @@
+                                   width=int(input("Enter width of your Rectangle:")))
+        print("The Area of Rectangle is:", rectangle)
+        break
+    elif finding.lower() == "triangle":
```
elif finding == "triangle":

kolyasalubov commented on HW7/Second_GitHub_Task.py:
```diff
@@ -0,0 +1,31 @@
+                                 base=int(input("Enter base of your Triangle:")))
+        print("The Area of Triangle is:", triangle)
+        break
+    elif finding.lower() == "circle":
```
elif finding == "circle":

kolyasalubov commented on HW7/Third_GitHub_Task.py:
```diff
@@ -0,0 +1,13 @@
+def numbers_chars(word: str) -> dict:
```
good job

kolyasalubov committed:
resolve conflict & create folder MykytaTitar
